tech_reports/TT-Distributed/TT-Distributed-Architecture-1219.md

```diff
@@ -1016,18 +1016,18 @@ Below, we include snippets from both the TT-Mesh and TT-Metal examples to illust
 MeshConfig mesh_config_0 = MeshConfig{.shape = virtual_mesh_shape, .offset = {0, 0}, .type=mesh_type};
 MeshConfig mesh_config_1 = MeshConfig{.shape = virtual_mesh_shape, .offset = {0, 4}, .type=mesh_type};
 
-DeviceHandle virtual_mesh_0 = CreateMeshDevice(mesh_config_0, 2 /* num_cqs */, DEFAULT_L1_SMALL_SIZE, DEFAULT_TRACE_REGION_SIZE, 1 /* num_command_queues */);
-DeviceHandle virtual_mesh_1 = CreateMeshDevice(mesh_config_1, 2 /* num_cqs */, DEFAULT_L1_SMALL_SIZE, DEFAULT_TRACE_REGION_SIZE, 1 /* num_command_queues */);
+DeviceHandle virtual_mesh_0 = CreateMeshDevice(mesh_config_0, DEFAULT_L1_SMALL_SIZE, DEFAULT_TRACE_REGION_SIZE, 1 /* num_command_queues */);
```

Comment:
`num_cqs` and `num_command_queues` were repeated. This version also aligns with the existing mesh device impl.